documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -0,0 +1,98 @@
+
+With the above points in mind, let's go for a practical example. Let's assume that our workload consists on an average
+ingestion rate of 1GBps and we have only 1 Stream in the system. Also, the serializers we use in our application have 
+a write amplification of 1.3x and we foresee an additional 5% of storage cost related to Pravega's metadata. Given 
+that, let's exercise how much Long-Term Storage capacity we would need with the following auto-retention policies:
+
+- _Size-based auto-retention of 1TB_: This policy attempts to keep at least 1TB of data related to our Stream.
+Therefore, the Long-Term Storage space needed would be at least: `1TB · 1.3 + 1TB · 0.05 = 1.35TB`. 
+
+- _Time-based retention of 1 day_: With this policy, Pravega will keep data for our Stream for at least 1 day.
+This means that our Long-Term Storage may require to store at least: `86400s * 1GBps * 1.3 + 86400s * 1GBps * 0.05 = 116.7TB`.
+
+Note that the above storage estimations are lower bounds. They assume that the Pravega auto-retention service immediately
+truncates the Stream once the retention policy has been met. However, the Pravega auto-retention service is implemented
+as a background task and should be properly configured under stringent retention requirements. Next, we provide some
+guidelines to configure the Pravega auto-retention service:
+
+- _Set a proper retention frequency_: By default, `controller.retention.frequency.minutes=30`. This means that Long-Term
+Storage should be provisioned to accommodate at least 30 minutes of data beyond what is configured in the auto-retention
+policy, until a Controller instance truncates the Stream. If your cluster should minimize the usage of Long-Term Storage
+resources, then you may want to set this parameter to a lower value (e.g., 1-5 minutes).
+
+- _Define the right number of buckets_: Similarly to what happens to Segment Containers, we need to think twice before
+setting the number of buckets. That is, if we leave the default value of 1, there would be only one Controller instance 
+handling all the work related to auto-retention. For this reason, we recommend to estimate the initial number of Controller 
+instances required in your Pravega Cluster: as a rule of thumb, having 1 to 2 Controllers every 3 Segment Stores is enough, 
+depending on the metadata load expected (e.g., working with small [Transactions](http://pravega.io/docs/latest/pravega-concepts/#transactions) 
+or with many Streams induces more metadata workload). Once the number of Controllers for a deployment is estimated,
+we can define a number of 4 to 8 times the number of Controllers. That is, if we have 3 Controller instances,
+we will configure `controller.retention.bucket.count=24`. With this, we could scale up the number of Controllers and
+the new instances will share the load along with with the previous ones.
+
+- _Controller retention thread pool size_: In general, having 1 thread (`controller.retention.thread.count=1`, default)
```

Comment:
We also need to talk about the Storage Rolling size. By default, we set it to a very large number (no rolling) so only deleting segments would reclaim LTS space. There's a config that we can set which defines the maximum size of a chunk in LTS, which should help reclaim space for mid-segment truncations.

Reply:
Added a mention to the parameter and the trade-off it represents.